src/app/store/effects/ReferenceData.effects.ts

```diff
@@ -0,0 +1,47 @@
+import { Injectable } from '@angular/core';
```

Comment:
I dont think you need these 2 effects for now. Because loading the store with preparers and testStation should just be the guard, action, reducer and selectors.